src/data/value/literal.rs

```diff
@@ -1,3 +1,4 @@
+use uuid::Uuid;
 use {
     super::{error::ValueError, Value},
     crate::{
```

Comment:
I think you can include it in use `{..}`

```suggestion
use {
    uuid::Uuid;
    super::{error::ValueError, Value},
    crate::{
```